UI/HelperFunctions/ShortlistFunctions.cs

```diff
@@ -0,0 +1,30 @@
+namespace UI.HelperFunctions;
+
+public static class ShortlistFunctions
```

Comment:
Good plan to move this in to a helper function - but not sure this is a "Shortlist" specific function?  It's used by various pages?

Reply:
You are right. The plan was to change the name but I forgot to change it - nice spot - thanks👍